x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ilm/LifecycleExecutionStateTests.java

```diff
@@ -145,6 +154,56 @@ public void testGetCurrentStepKey() {
         assertNull(error6.getMessage());
     }
 
+    public void testPotentiallyTruncateLongJsonWithExplanationNotTruncated() {
```

Comment:
Can you add one-sentence explanations to these tests to explain what they test exactly? Especially the ones below have pretty minor differences so it's hard to tell what each one is testing exactly?